src/main/webapp/help-jsonParam.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    (Optional) Specify any additional parametrs in JSON format.
+    <strong> Note:- If the same parameter from above form is provided in JSON the form parameter takes preference and
```

Comment:
`Note:- If the same parameter is specified in another field, that other value will take precedence.`